src/contexts/PreferencesContext.tsx

```diff
@@ -93,10 +93,13 @@ export const PreferencesProvider = ({
       ).then(response => response.json());
       return data?.value;
     } catch (error) {
-      console.log(
-        `Potential error fetching preferences, or preference with key ${key} is not set:`,
-        error
-      );
+      if (error instanceof HTTPError && error.responseCode === 404) {
+        console.log(`Preference '${key}' not found`);
```

Comment:
This hides the stack traces which used to be thrown when there are no preferences.